packages/rum-core/src/domain/view/viewMetrics/trackCumulativeLayoutShift.spec.ts

```diff
@@ -215,31 +250,39 @@ describe('trackCumulativeLayoutShift', () => {
       expect(clsCallback.calls.mostRecent().args[0].targetSelector).toEqual(undefined)
     })
 
-    it('should get the target element of the largest layout shift', () => {
-      const { lifeCycle, clock } = setupBuilder.withFakeClock().build()
+    it('should get the target element and time of the largest layout shift', () => {
+      const { lifeCycle } = setupBuilder.build()
       const divElement = appendElement('<div id="div-element"></div>')
 
+      // first session window:  { value: 0.5, time: 1, targetSelector: '#div-element' }
       lifeCycle.notify(LifeCycleEventType.PERFORMANCE_ENTRIES_COLLECTED, [
-        createPerformanceEntry(RumPerformanceEntryType.LAYOUT_SHIFT, { value: 0.1 }),
+        createPerformanceEntry(RumPerformanceEntryType.LAYOUT_SHIFT, { value: 0.1, startTime: 0 as RelativeTime }),
       ])
       lifeCycle.notify(LifeCycleEventType.PERFORMANCE_ENTRIES_COLLECTED, [
-        createPerformanceEntry(RumPerformanceEntryType.LAYOUT_SHIFT, { value: 0.2, sources: [{ node: divElement }] }),
+        createPerformanceEntry(RumPerformanceEntryType.LAYOUT_SHIFT, {
+          value: 0.2,
+          startTime: 1 as RelativeTime,
+          sources: [{ node: divElement }],
+        }),
       ])
       lifeCycle.notify(LifeCycleEventType.PERFORMANCE_ENTRIES_COLLECTED, [
-        createPerformanceEntry(RumPerformanceEntryType.LAYOUT_SHIFT, { value: 0.2 }),
+        createPerformanceEntry(RumPerformanceEntryType.LAYOUT_SHIFT, { value: 0.2, startTime: 2 as RelativeTime }),
       ])
 
-      // second session window
-      clock.tick(MAX_WINDOW_DURATION + 1)
+      // second session window:  { value: 0.4, time: 5002, targetSelector: undefined }
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏ Why not keep using MAX_WINDOW_DURATION. It helps to understand why it creates a new window.